qml/pages/ThreadView.qml

```diff
@@ -569,6 +592,39 @@ Page {
                         }
                     }
                 }
+                Flow { id: pollsItem
+                    visible: index == 0 && has_polls
+                    property int cols: 3
+                    width: parent.width
+                    Label { id: pollHeader
+                        width: parent.width
+                        text: (has_polls > 1 )
+                            ? qsTr("This post contains polls.")
+                            : qsTr("This post contains a poll.")
+                            + " " + qsTr("Click to view and vote:")
+                        font.pixelSize: Theme.fontSizeSmall
+                        color: Theme.highlightColor
+                    }
+                    Repeater {
+                        model: has_polls
+                        delegate: ValueButton { id: pollButt
+                            property int pollindex: index
+                            onClicked: {
+                                const pd = polldata.get(pollindex)
+                                console.debug("Opening poll no", pollindex, "for post", postid) //, ", data:", JSON.stringify(pd,null,2))
+                                pageStack.push("PollView.qml",
+                                    { "key": loggedin.value, "postid": postid,
+                                      "polldata": pd["poll"],
+                                      "submitted_votes":  ((loggedin.value != "-1" ) && pd["votes"]) ? pd["votes"]["list"] : []
```

Comment:
Can just blindly pass pd["votes"]["list"] here if else is added at 296, otherwise was getting undefined and poll would always open as submitted